space_packet_parser/xtce/parameter_types.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, name: str, encoding: encodings.DataEncoding, unit: Optional[s
         self.encoding = encoding
         self.unit = unit
 
+    def __hash__(self) -> int:
+        return hash(self.name)
```

Comment:
Do we need these hashes redefined? The keys in the dictionary lookups are still string names, so the dictionary hash is this in a sense. i.e. I'm not sure we are using the object hashes anywhere?

Reply:
Oh, thanks for catching that. I had these in when I was planning to implement internal `set` objects but I ditched that idea when I saw the resulting performance hit for doing lookups.

Reply:
No hash method does mean that we cannot create set objects from these classes but that's not really necessary currently.